data/migrations/20190812100226_create-cars.js

```diff
@@ -0,0 +1,19 @@
+
+exports.up = function(knex) {
+    return knex.schema.createTable('users', tbl => {
```

Comment:
is the table name appropriate for the data you are handling? You can call it whatever you want as long as it is consistent, but it would probably be better practice to describe what it is.